docs/download/index.md

```diff
@@ -15,15 +15,11 @@ License & terms of use: http://www.unicode.org/copyright.html
 If you want to use ICU (as opposed to developing it), it is recommended that you download an official packaged version of the ICU source code. These versions are tested more thoroughly than day-to-day development builds of the system, and they are packaged in zip and tar files for convenient download. Here are several recent releases of ICU that are available:
 
 ## Release Candidate
-***2024-09-30: ICU 76 release candidate is now available.***
+***2024-10-24: ICU 76 is now available.***
 It updates to [Unicode 16](https://www.unicode.org/versions/Unicode16.0.0/) ([blog](https://blog.unicode.org/2024/09/announcing-unicode-standard-version-160.html)), including new characters and scripts, emoji, collation & IDNA changes, and corresponding APIs and implementations. It also updates to [CLDR 46](https://cldr.unicode.org/downloads/cldr-46) ([beta blog](https://blog.unicode.org/2024/09/unicode-cldr-46-beta-available-for.html)) locale data with new locales, significant updates to existing locales, and various additions and corrections. For example, the CLDR and Unicode default sort orders are now very nearly the same.
 
 Most of the java.time (Temporal) types can now be formatted directly. There are some new APIs to make ICU easier to use with modern C++ and Java patterns. The Java and C++ technology preview implementations of the CLDR MessageFormat 2.0 specification have been updated to match recent changes. See [ICU 76](76.md).
 
-Please test this release candidate on your platforms and report bugs and regressions by Monday, 2024-oct-21, via the [icu-support](https://icu.unicode.org/contacts) mailing list, and/or please [find/submit error reports](https://icu.unicode.org/bugs).
-
-Please do not use this release candidate in production.
-
 ## Latest Release
```

Comment:
"## Previous Release"
?

Reply:
When people go to the general ICU download page, we should always point them to the latest release.

Maybe that means that the description for the ICU 76 release candidate moves into the Latest Release section, and the description for ICU 75 moves into the Previous Releases section?